Its.Log.Monitoring/Test.cs

```diff
@@ -16,5 +15,8 @@ internal class Test
 
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public string[] Tags { get; set; }
+
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public TestParameter[] QueryParameters { get; set; }

```

Comment:
The mismatch between the type name and the property name is a little distracting. Why not 

`public TestParameter[] QueryParameters`

or

`public QueryParameter[] QueryParameters`

or

`public TestParameter[] TestParameters`

or just 

`public Parameter[] Parameters`